nexus/tests/output/uncovered-authz-endpoints.txt

```diff
@@ -7,3 +7,16 @@ login_spoof                              (post   "/login")
 login_local                              (post   "/login/{silo_name}/local")
 login_saml                               (post   "/login/{silo_name}/saml/{provider_name}")
 logout                                   (post   "/logout")
+
+Deprecated API endpoints to be removed at the end of the V1 migration
```

Comment:
Because I updated the test _not_ to be an exact file match we can add extra lines of text in here.